### PR TITLE
Adds November Standard for Public Code community call summary

### LINKED ISSUE #461: Make blog post for November Standard community call
Aim to release today or tomorrow.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #461

Intended to be published today.

### CODE REVIEW

**Review by ericherman — APPROVED** [review]

File: _posts/2023-11-06-notes-from-community-call-2-november-2023.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### Background
+
+We have long thought that to be able to say that the Standard for Public Code is production ready, it should be in use for at least a couple of codebases that meet all the criteria. This would validate that the requirements are proven in practice to be good.
+We have talked about this in a [previous community call]({% post_url 2022-07-07-notes-from-community-call-7-july-2022 %}) and made a [note in the roadmap](https://github.com/publiccodenet/standard/blob/develop/docs/roadmap.md) about it.
+When talking about applying to become an ISO standard in [last month's community call]({% post_url 2023-10-09-notes-from-community-call-5-october-2023 %}), this naturally became worth reviewing again.
+
+### Notes
+
+First we talked a bit about what maturity might mean for a standard and when it could be too early to call it production ready.
+As mentioned in the background, we aimed for it to be used by at few codebases first, and formalizing that in an ambition might radiate proper intent for others.
```
Comment (ElenaFdR) [inline _posts/2023-11-06-notes-from-community-call-2-november-2023.md:40]:
```suggestion
As mentioned in the background, we aimed for it to be used by at few codebases first, and formalizing that in an ambition might radiate proper intent to others.
```

File: _posts/2023-11-06-notes-from-community-call-2-november-2023.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+It was also suggested that, ideally, one would even leave some time after the codebases fully comply without requirements being changed or repealed to ensure that they were well enough formulated.
```
Comment (ElenaFdR) [inline _posts/2023-11-06-notes-from-community-call-2-november-2023.md:41]:
```suggestion
It was also suggested that, ideally, one would even leave some time after the codebases fully comply without requirements being changed or revoked to ensure that they were formulated well enough .
```

File: _posts/2023-11-06-notes-from-community-call-2-november-2023.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Secondly, we came in to a meta question of why even aim for a version 1.0 release.
+While the number is not critical per se, it is a strong signal, especially when adhering to semantic versioning, that it is a product ready to be used.
```
Comment (ElenaFdR) [inline _posts/2023-11-06-notes-from-community-call-2-november-2023.md:44]:
```suggestion
While the number is not critical per se, it is a strong signal, especially when adhering to semantic versioning, that a product is ready to be used.
```

File: _posts/2023-11-06-notes-from-community-call-2-november-2023.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+So there is no external pressure, but more of a wish from the community to be clear about what stage the Standard for Public Code is at.
+This also led into a subdiscussion about how to apply semantic versioning for a standard in terms of defining what its public API is.
+We think this aspect is already addressed, but may follow up with more documentation on our thoughts about that, and if it seems to have some general value, also publish a blog post on that topic.
```
Comment (ElenaFdR) [inline _posts/2023-11-06-notes-from-community-call-2-november-2023.md:47]:
```suggestion
We think this aspect is already addressed, but may follow up with more documentation on our thoughts about that, and if it seems to have some general value, also publish a blog post.
```

File: _posts/2023-11-06-notes-from-community-call-2-november-2023.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Another aspect we discussed was more technical thresholds for moving it into 1.0.
+For example, it might be useful to look at a 1.0 release in terms of what we *don't* want in there.
+By that we meant listing criteria as; the standard should have no critical issues, or, there should be no requirements conflicting with each other etc.
```
Comment (ElenaFdR) [inline _posts/2023-11-06-notes-from-community-call-2-november-2023.md:51]:
```suggestion
By that we meant listing criteria like 'the standard should have no critical issues', or 'there should be no requirements conflicting with each other', etc.
```

File: _posts/2023-11-06-notes-from-community-call-2-november-2023.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+1. The ability to "deep link" directly to a requirement. We work on that in [issue 327](https://github.com/publiccodenet/standard/issues/327) but also got a tip to look at legislative linking standards like the [European Legislation Identifier](https://en.wikipedia.org/wiki/European_Legislation_Identifier) (ELI) and the blog post [A love letter to the Parliamentary Counsel of the world]( https://hamish.dev/a-love-letter-to-the-parliamentary-counsel-of-the-world) which has some extensive writing on the topic. Our aim for now is that deep links like these should be stable over time so that we can avoid breaking links even if requirements are reordered or deleted.
+2. The ability to find older versions on the website. Currently, older versions are available on GitHub, but they are not easy to read or to link to. Later, when we have several versions after 1.0 and especially if some of them are ISO standards, it will be more important to be able to refer to a criterion of a specific version of the standard.
+
+Finally, we had a brief discussion if the benefits of getting the standard to become an ISO standard outweigh the added overhead.
```
Comment (ElenaFdR) [inline _posts/2023-11-06-notes-from-community-call-2-november-2023.md:61]:
```suggestion
Finally, we briefly discussed if the benefits of getting the standard to become an ISO standard outweigh the added overhead.
```

**Review by ElenaFdR — COMMENTED** [review]
I've made a few more language edits - sorry for not catching these on my first pass!

**Pull request merged**